src/style.css

```diff
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+.page .page__card {
```

Comment:
Unnecessary class `.page` used. Check BEM checklist for explanation.

Reply:
Agree, don't increase selectors specificity